src/System.Management.Automation/engine/hostifaces/HostUtilities.cs

```diff
@@ -69,7 +69,7 @@ public static class HostUtilities
             [System.Diagnostics.DebuggerHidden()]
             param([string] $formatString)
 
-            $formatString -f [string]::Join(', ', (Get-Command $lastError.TargetObject -UseFuzzyMatch | Select-Object -First 10 -Unique -ExpandProperty Name))
+            $formatString -f [string]::Join(', ', (Get-Command $lastError.TargetObject -UseFuzzyMatching -FuzzyMinimumDistance 1 | Select-Object -First 5 -Unique -ExpandProperty Name))
```

Comment:
The `FeedbackProvider` implementation is an experimental feature, so when it's disabled, we still need this and its related code here in this file in case the `FuzzySuggestion` experimental feature is enabled.

Once the `FeedbackProvider` is made stable (by the end of 7.4 I guess?), we will remove all the old suggestion related code in this file.